application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -133,35 +133,36 @@ private boolean isUserOnCooldown(User user) {
     }
 
     private void sendCooldownResponse(SlashCommandInteractionEvent event) {
+        Guild guild = event.getGuild();
         User user = event.getUser();
 
         String message =
                 """
-                        Sorry, you can only create a single help thread every 5 minutes. Please use your existing thread %s instead.
-                        If you made a typo or similar, you can adjust the title using the command %s and the category with %s :ok_hand:""";
-
-        RestAction<String> changeTitle = MessageUtils.mentionGuildSlashCommand(event.getGuild(),
-                HelpThreadCommand.COMMAND_NAME, HelpThreadCommand.CHANGE_SUBCOMMAND_GROUP,
-                HelpThreadCommand.CHANGE_TITLE_SUBCOMMAND);
-        RestAction<String> changeCategory = MessageUtils.mentionGuildSlashCommand(event.getGuild(),
-                HelpThreadCommand.COMMAND_NAME, HelpThreadCommand.CHANGE_SUBCOMMAND_GROUP,
-                HelpThreadCommand.CHANGE_CATEGORY_OPTION);
-        long lastCreatedThreadId = database
+                        Sorry, you can only create a single help thread every %s %s. Please use your existing thread %s instead.
```

Comment:
the text should be closer to where its used. right now its quite far away and then its easy to get confused with the parameters.

Reply:
done